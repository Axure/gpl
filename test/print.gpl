void main() {
    print(56 + 4*2 - 7);
    print('w');
    print("I'm a string:)");
}

