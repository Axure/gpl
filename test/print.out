57
w
I'm a string:)
